src/components/menus/Items.ts

```diff
@@ -111,9 +112,80 @@ export class Items<TGameStartr extends FullScreenPokemon> extends Component<TGam
      * @param settings   Custom attributes to apply to the menu.
      */
     public openItemMenu(listing: IInventoryListing, settings: IItemMenuSettings): void {
-        const options: any[] = [
+        let options: any[] = [];
+        if (this.gameStarter.features.heldItems) {
+            options = [
             {
                 callback: (): void => {
+                    const category = this.gameStarter.itemsHolder.getItem(listing.item).category;
+                    if (category === "HM") {
+                        { return; }
+                    }
+                    if (category === "TM") {
```

Comment:
If you do need to return, combine the if statements.

What's with the extra `{` and `}`?